doc/reference/index.rst

```diff
@@ -0,0 +1,9 @@
+.. _reference:
+
+Reference
```

Comment:
As a top level now, this might be better called "Configuration"

Reply:
i would rather leave as reference, because that what it is. We can add other reference type sections there. Configuration might be misleading, there is nothing about configuration in there.